apps/retrospective/services/sprint_task_member.go

```diff
@@ -63,6 +63,34 @@ func (service SprintTaskMemberService) GetMembers(
 	return members, http.StatusOK, nil
 }
 
+// AddSprintActiveMember ...
+func (service SprintTaskMemberService) AddSprintActiveMember(taskList *retroSerializers.SprintTasksSerializer,
+	memberID uint,
+	retroID string,
+	sprintID string) *retroSerializers.SprintTasksSerializer {
+	db := service.DB
+	sprintMemberTask := retroModels.SprintMemberTask{}
+	sprintMember := retroModels.SprintMember{}
+
+	db.Model(&retroModels.SprintMember{}).
+		Where("sprint_members.deleted_at IS NULL").
```

Comment:
Please use the `NotDeletedScope` here and create it if not created now.

Reply:
removed this